ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/loopback-interfaces.j2

```diff
@@ -1,50 +1,50 @@
 {# eos- Loopback Interfaces #}
-{% if loopback_interfaces is defined and loopback_interfaces is not none %}
+{% if loopback_interfaces is arista.avd.defined %}
```

Comment:
As `arista.avd.defined` is included in `arista.avd.natural_sort`, if statement can be removed

Reply:
this check is redundant and should be removed